sherlock/resources/data.json

```diff
@@ -1246,6 +1246,13 @@
     "username_claimed": "Matt-Riggsby",
     "username_unclaimed": "noonewouldeverusethis7"
   },
+  "Quizlet": {
+    "errorType": "Page Unavailable",
```

Comment:
Looks like you made a small mistake here. `errorType` should be `message` and `Page Unavailable` should be assigned to `errorMsg`